src/types/events/base-events.ts

```diff
@@ -651,6 +651,7 @@ export interface ReactionMessageItem {
   ts: string;
 }
 
+// This type is deprecated.
```

Comment:
Maybe we can annotate this with [JSDoc to better inform about the deprecation](https://jsdoc.app/tags-deprecated.html)?

```suggestion
/**
 * @deprecated Slack applications no longer receive reaction events for files and file comments.
 */
```